### PR TITLE
MAISTRA-1825 fix prometheus integration; MAISTRA-1804 specify istio c…

### PR DESCRIPTION
**Pull request by rcernich**
…onfig map name for kiali

Signed-off-by: rcernich <rcernich@redhat.com>

### CODE REVIEW

**Comment by dgn** [comment]
I think I'm lacking context here. EnvoyFilter resources in the control plane namespace apply to all mesh namespaces- so why do we copy them around?

File: pkg/controller/servicemesh/memberroll/namespace_reconciler.go [hunk 1]
```diff
@@ -326,6 +339,65 @@ func (r *namespaceReconciler) reconcileRoleBindings(ctx context.Context, namespa
...
+	labelSelector := map[string]string{common.MemberOfKey: r.meshNamespace}
+	err := r.Client.List(ctx, &namespaceEnvoyFilters, client.MatchingLabels(labelSelector), client.InNamespace(namespace))
+	if err != nil {
+		reqLogger.Error(err, "error retrieving EnvoyFilter resources for namespace")
+		return err
+	}
+
+	allErrors := []error{}
+
+	// add required role bindings
```
Comment (dgn) [inline pkg/controller/servicemesh/memberroll/namespace_reconciler.go:355]: nit: envoy filters

**Comment by rcernich** [comment]
Yep.  I need to rework this.  The problem is that the version is not being set correctly in the proxy, so the filters aren't being applied.

**Comment by rcernich** [comment]
Superseded by https://github.com/maistra/istio-operator/pull/555

**Pull request closed without merging**